src/script/meson.build

```diff
@@ -9,8 +9,12 @@ foreach s : [
     'global_options.meson',
     'per_project_options.meson',
     'vcs_tagger.meson',
+    'gettext.meson',
```

Comment:
I think that some kind of tree structure to organise scripts is absolutely necessary to avoid things getting out of hand. I don't have much of an opinion on the structure though beyond the conviction that the current flat structure does not scale.